### PR TITLE
add delta bundle comment in the UT test

### PR DESCRIPTION
**Pull request by yanmxa**
Signed-off-by: myan <myan@redhat.com>

### CODE REVIEW

File: manager/pkg/statussyncer/transport2db/syncer/dbsyncer/policies_db_syncer_test.go [hunk 1]
```diff
@@ -251,16 +252,19 @@ var _ = Describe("Policies", Ordered, func() {
...
+		// id(d9347b09-bb46-4e2b-91ea-513e83ab9ea7) hub1-cluster2 compliant
 		transportPayload.Objects = append(transportPayload.Objects, &status.PolicyGenericComplianceStatus{
 			PolicyID:                  createdPolicyId,
 			CompliantClusters:         []string{"cluster1"},
 			NonCompliantClusters:      []string{"cluster3"},
 			UnknownComplianceClusters: make([]string, 0),
 		})
+		// expect:
+		// id(d9347b09-bb46-4e2b-91ea-513e83ab9ea7) hub1-cluster1 compliant
+		// id(d9347b09-bb46-4e2b-91ea-513e83ab9ea7) hub1-cluster2 compliant
```
Comment (clyang82) [inline manager/pkg/statussyncer/transport2db/syncer/dbsyncer/policies_db_syncer_test.go:266]: `hub1-cluster3` non_compliant?

Comment (clyang82) [reply manager/pkg/statussyncer/transport2db/syncer/dbsyncer/policies_db_syncer_test.go:266]: Does it mean that if there is no record in database, it won't insert into?

Comment (yanmxa) [reply manager/pkg/statussyncer/transport2db/syncer/dbsyncer/policies_db_syncer_test.go:266]: Yes, this conclusion is based on the results of my tests so far.

**Pull request merged**